packages/core-scroll/core-scroll.js

```diff
@@ -77,10 +77,25 @@ export default class CoreScroll extends HTMLElement {
     window.addEventListener('load', this) // Update state when we are sure all CSS is loaded
     document.addEventListener('click', this)
     setTimeout(() => this.handleEvent()) // Initialize buttons after children is parsed
+
+    // Initialize buttons when children change
+    // - jsx relies on onScrollChange triggering to update button states
+    if (!this._childListObserver && window.MutationObserver) {
+      this._childListObserver = new window.MutationObserver((mutationList) => {
+        for (const mutation of mutationList) {
+          if (mutation.type === 'childList') {
+            this.handleEvent()
+          }
+        }
+      })
+    }
+    this._childListObserver.observe(this, { childList: true, subtree: true })
   }
 
   disconnectedCallback () {
-    this._throttledEvent = null // Garbage collection
+    if (this._childListObserver) this._childListObserver.disconnect()
+    this._childListObserver = null // Garbage collection
+    this._throttledEvent = null
```

Comment:
🏌️ here, but for brevity lines 97+98 can be merged to `this._childListObserver = this._throttledEvent = null // Garbage collection` similar to how we do in other components.